docs/collections/_schema/human-readable-schema.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: default
+title: Human-Readable schema format
+nav_order: 2
+---
+
+# Human-Readable schema format {#schema}
+{: .no_toc }
+
+<details open markdown="block">
+  <summary>
+    Topics on this page
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+This topic describes Cedar's human-readable schema format.
+
+## Schema format {#schema-format}
+
+A schema consists of zero or more namespaces, each of which contains declarations of three types --- *Entity Type Declaration*, *Action Declaration*, and *Common Type Declaration*. These declarations define entity types, actions, and common types used to define the former two, respectively. Declarations are delimited by `;`s. Note that unlike the JSON schema format, the human-readable schema format allows you to write Cedar-style comments.
+
+## NameSpace {#schema-namespace}
+
+You can group the declarations of a namespace by wrapping them with curly braces like `namespace Foo {...}`. A name must be given for a namespace specified by such syntax. An alternative way to declare a namespace is to “inline” declarations like `entity Bar; namespace Foo {...}`, where the entity declaration of `Bar` is under the same scope of namespace `Foo`. Names of these declarations are always referred as they are whereas declaration names of a `namespace` construct must be referred as their fully-qualified forms in other namespaces.
```

Comment:
I found the wording here confusing. Maybe:

---

You can create a namespace with associated declarations using the `namespace` key word like `namespace Foo { entity Bar; }`. The name for a namespace must be an identifier as specified in Cedar's syntax. Anything declared in this namespace must be referred to in its fully-qualified form when referenced outside of the namespace, so the declared entity type would be `Foo::Bar`.

You can instead place declaration directly outside of any namespace like `entity Bar;`. Names of these declarations are always referred to without qualification.

Reply:
Thanks John. Worked up a bit more and added to revision draft.